src/app/Views/species_search.php

```diff
@@ -3,100 +3,102 @@
+				<input class="form-check-input" type="radio" name="species-group"  id="plants"
+					value="plants" <?php echo set_radio('groups', 'plants', true); ?> />
+				<label class="form-check-label" for="scientific-name">
+					only plants
+				</label>
+			</div>
+			<div class="form-check form-check-inline">
+				<input class="form-check-input" type="radio" name="species-group"  id="bryophytes"
+					value="bryophytes" <?php echo set_radio('groups', 'bryophytes'); ?> />
+				<label class="form-check-label" for="axiophyte-name">
+					only bryophytes
+				</label>
+			</div>
+			<div class="form-check form-check-inline">
+				<input class="form-check-input" type="radio" name="species-group" id="both"
+					value="both" <?php echo set_radio('groups', 'both'); ?> />
+				<label class="form-check-label" for="common-name">
+					both plants and bryophytes
+				</label>
+			</div>
+		</div>
+	</div>
 <?php echo form_close() ?>
 <!-- Show the search results if there are any -->
 <?php if (isset($speciesList)):?>
-    <table class="table">
-        <thead><tr>
-            <th class="d-none d-md-table-cell">Family</th>
-            <th>Scientific Name</th>
-            <th class="d-none d-sm-table-cell">Common Name</th>
-            <th>Records</th>
-        </tr></thead>
-        <tbody>
-        <?php foreach ($speciesList as $species):?>
-        <tr>
-            <td class="d-none d-md-table-cell"><?php echo $species->family?></td>
-            <td><a href="<?php echo base_url("/species/{$species->name}");?>"><?=$species->name?></a></td>
-            <td class="d-none d-sm-table-cell"><a href="<?php echo base_url("/species/{$species->name}");?>"><?php echo $species->commonName?></a></td>
-            <td><?=$species->count?></td>
-        </tr>
-        <?php endforeach;?>
-        </tbody>
-    </table>
-    <nav>
-        <ul class="pagination justify-content-center">
-            <li class="page-item"><a class="page-link" href="#">Previous</a></li>
-            <li class="page-item"><a class="page-link" href="#">1</a></li>
-            <li class="page-item"><a class="page-link" href="#">2</a></li>
-            <li class="page-item"><a class="page-link" href="#">3</a></li>
-            <li class="page-item"><a class="page-link" href="#">Next</a></li>
-        </ul>
-    </nav>
+	<table class="table">
+		<thead><tr>
+			<th class="d-none d-md-table-cell">Family</th>
+			<th>Scientific Name</th>
+			<th class="d-none d-sm-table-cell">Common Name</th>
+			<th>Records</th>
+		</tr></thead>
+		<tbody>
+		<?php foreach ($speciesList as $species):?>
+		<tr>
+			<td class="d-none d-md-table-cell"><?php echo $species->family?></td>
+			<td><a href="<?php echo base_url('/species/' . $species->name); ?>"><?php echo $species->name ?></a></td>
+			<td class="d-none d-sm-table-cell">
+				<a href="<?php echo base_url('/species/' . $species->name); ?>"><?php echo $species->commonName ?></a>
+			</td>
+			<td><?php echo $species->count ?></td>
```

Comment:
PHPCS didn't like the string interpolation style of doing things here and wanted concatenation instead.

Also changed a couple of instances of `<?=` to `<?php echo` for consistency.

Reply:
I wonder if the single-quote + concatenation approach is recommended because it helps distinguish (visually) from the surrounding HTML attribute quotes? Looks like it could be.